### DISCUSSION TITLE
Custom sync inverval with google tasks and calendar

### DISCUSSION THREAD

**Discussion by crestcere** [kewisch/gdata-provider#436] [Support] [answered]
I want to sync my google tasks for every 1, 5, 10, and 15 minutes but I can’t manage to do that.

Only 30 and 60 mins shown in the options menu and changing “calendar.registry.[UUID].refreshInterval” values belonging to this addon in the config editor do not work either. It keeps resetting its value and does not affect.

Is there any way to accomplish that? I can change the sync interval with google calendar which is added with default option.

**Comment by kewisch** [comment, marked as answer]
Unfortunately the longer interval was necessary to avoid triggering rate limiting. I'd love to support a shorter interval, though I am afraid that this would make the provide unusable for everyone.
